opencensus/trace/span_context.py

```diff
@@ -167,5 +167,6 @@ def generate_trace_id():
     :rtype: str
     :returns: 32 digit randomly generated hex trace id.
     """
-    trace_id = uuid.uuid4().hex
-    return trace_id
+    t = int(time.time())
+    lower_96 = random.getrandbits(96)
+    return '{:032x}'.format((t << 96) | lower_96)
```

Comment:
Curious, why would we want to encode the time in the high bits? There could be side effect if sampler is using the high bits.
@bogdandrutu @songy23 Do we have a cross language guideline on the randomness of trace_id and span_id? (and how samplers should do)

Reply:
Zipkin does this for interop with Amazon X-Ray. See the notes here: https://github.com/apache/incubator-zipkin/issues/1754